### PR TITLE
prepare for release v1

### PR DESCRIPTION
**Pull request by robvet**
## Summary

Final configuration for v1 release:

1. Add "send your feedback" like in Blazor book
2. Remove Preview text from all files
3. Add edition text - like the microservices book. 
4. Replace book image with 1.0 
5. Update Copyright Year 2020
6. Update Architecture index page - Remove "(preview)"
7. Update ToC - Move books out of Preview

### CODE REVIEW

File: docs/architecture/cloud-native/index.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+ms.date: 05/13/2020
 ---
 
 # Architecting Cloud Native .NET Applications for Azure
 
-[!INCLUDE [book-preview](../../../includes/book-preview.md)]
-
 ![cover image](./media/cover.png)
 
+**EDITION v.1
```
Comment (nishanil) [inline docs/architecture/cloud-native/index.md:12]:
```suggestion
**EDITION v.1.0**
```

**Comment by nishanil** [comment]
@robvet Since the Version 1.0 is updated to align with .NET Core 3.1, update the code snippets and images to the same wherever it is referenced. Example: https://review.docs.microsoft.com/en-us/dotnet/architecture/cloud-native/leverage-containers-orchestrators?branch=pr-en-us-18429#visual-studio-docker-tooling

**Comment by robvet** [comment]
Replaced Figure 3-8 and 3-9
Removed eShop reference from index.md
Brief wordsmithing for Ch 1
Updated images in Ch 1 and 3

File: docs/architecture/cloud-native/index.md [hunk 2]
```diff
@@ -59,6 +59,10 @@ Editors:
 
 > **Maira Wenzel**, Program Manager, .NET team, Microsoft
 
+## Version
```
Comment (nishanil) [inline docs/architecture/cloud-native/index.md:62]: remove the text on Line 56: `Learn more about eShopOnContainers` Looks like a merge issue?

File: docs/architecture/cloud-native/infrastructure-resiliency-azure.md [hunk 3]
```diff
@@ -51,19 +65,19 @@ The cloud thrives on scaling. The ability to increase/decrease system resources
...
 
 - *Take advantage of platform autoscaling features.* Use built-in autoscaling features whenever possible, rather than custom or third-party mechanisms. Where possible, use scheduled scaling rules to ensure that resources are available without a startup delay, but add reactive autoscaling to the rules as appropriate, to cope with unexpected changes in demand. For more information, see [Autoscaling guidance](https://docs.microsoft.com/azure/architecture/best-practices/auto-scaling).
 
-- *Scale-out aggressively.* A final practice would be to scale out aggressively so that you can quickly meet immediate spikes in traffic without losing business. And, then scale in (that is, remove unneeded instances) conservatively to keep the system stable. A simple way to implement this is to set the cool down period, which is the time to wait between scaling operations, to five minutes for adding resources and up to 15 minutes for removing instances.
```
Comment (nishanil) [inline docs/architecture/cloud-native/infrastructure-resiliency-azure.md:66]: Why was the casing changed?

File: docs/architecture/cloud-native/infrastructure-resiliency-azure.md [hunk 4]
```diff
@@ -73,7 +87,7 @@ We encouraged the best practice of implementing programmatic retry operations in
 
 - *Azure Redis Cache.* The Redis StackExchange client uses a connection manager class that includes retries on failed attempts. The number of retries, specific retry policy and wait time are all configurable.
 
-- *Azure Service Bus.* The Service Bus client exposes a [RetryPolicy class](xref:Microsoft.ServiceBus.RetryPolicy) that can be configured with a back-off interval, retry count, and <xref:Microsoft.ServiceBus.RetryExponential.TerminationTimeBuffer%2A>, which specifies the maximum time an operation can take. The default policy is nine maximum retry attempts with a 30-second backoff period between attempts.
+- *Azure Service Bus.* The Service Bus client exposes a [RetryPolicy class](https://docs.microsoft.com/dotnet/api/microsoft.servicebus.retrypolicy?view=azure-dotnet) that can be configured with a back-off interval and retry count, which specifies the maximum time an operation can take. The default policy is nine maximum retry attempts with a 30-second backoff period between attempts.
```
Comment (nishanil) [inline docs/architecture/cloud-native/infrastructure-resiliency-azure.md:90]: The earlier reference is correct. Please revert to original.

File: docs/architecture/cloud-native/resiliency.md [hunk 5]
```diff
@@ -1,39 +1,41 @@
...
+author: robvet
+ms.date: 05/13/2020
 ---
+
 # Cloud-native resiliency
 
-[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+Resiliency is the ability of your system to react to failure and still remain functional. It's not about avoiding failure, but accepting failure as inevitable and crafting  your cloud-native services to respond to it. The end-goal is to return the application to a fully functioning state after a failure.
+
+Cloud-native systems take full advantage of the cloud computing model. Your services must know how to thrive in a dynamic, virtualized cloud environment. They'll make extensive use of Platform as a Service (PaaS) compute infrastructure and managed services. The caveat: Cloud infrastructure and networking are inherently unreliable.
```
Comment (nishanil) [inline docs/architecture/cloud-native/resiliency.md:12]: inherently unreliable is a strong statement and the statement when taken out of context sounds very negative. Can you rephrase to emphasize on transient failures than calling it unreliable? Also, talk about why it is important in a distributed apps to mitigate cascading failures.

File: docs/architecture/cloud-native/resiliency.md [hunk 6]
```diff
@@ -1,39 +1,41 @@
...
 
-Resiliency is the ability of your system to react to failure and still remain functional. It isn't about avoiding failure. But it's about accepting that failure is inevitable in cloud-based systems and building your application to respond to it. The end-goal of resiliency is to return the application to a fully functioning state after a failure.
+Cloud vendors can detect and mitigate platform failure. Your service will be restarted, scale out and in, and distributed across nodes. Your services must be design services to detect these environmental scenarios and recover.
```
Comment (nishanil) [inline docs/architecture/cloud-native/resiliency.md:14]: Rephrase this sentence - _Your services must be design services.._ - doesn't read right.

File: docs/architecture/cloud-native/resiliency.md (hunk 5 above)
Comment (nishanil) [reply docs/architecture/cloud-native/resiliency.md:12]: I felt very disconnected reading the intro.

File: docs/architecture/cloud-native/resilient-communications.md [hunk 7]
```diff
@@ -1,54 +1,54 @@
...
+A better approach is an evolving technology entitled *Service Mesh*. A [service mesh](https://www.nginx.com/blog/what-is-a-service-mesh/) is a configurable infrastructure layer with built-in capabilities to handle service communication and the other challenges mentioned above. It decouples these concerns by moving them into a service mesh proxy. Known as the [Sidecar pattern](https://docs.microsoft.com/azure/architecture/patterns/sidecar), the proxy is deployed into a separate process to provide isolation from your service business code. However, the proxy is closely linked to the service. It is created along with it and shares its lifecycle. Figure 6-7 shows this scenario.
```
Comment (nishanil) [inline docs/architecture/cloud-native/resilient-communications.md:23]: What's service mesh proxy? Shouldn't it be called a sidecar proxy?

**Review by nishanil — COMMENTED** [review]
@robvet  Left a few comments for you to address. Also, address these mentioned below.
- Figure 3-5 and 3-6 need to be  updated with .NET Core 3.1. 3-5 needs a new image and 3-6 requires to update the code 
- drop the file cover-old.png

File: docs/architecture/cloud-native/index.md (hunk 2 above)
Comment (robvet) [reply docs/architecture/cloud-native/index.md:62]: Darn, thought I got rid of this. Gone now.

File: docs/architecture/cloud-native/infrastructure-resiliency-azure.md (hunk 3 above)
Comment (robvet) [reply docs/architecture/cloud-native/infrastructure-resiliency-azure.md:66]: Fixed

File: docs/architecture/cloud-native/infrastructure-resiliency-azure.md (hunk 4 above)
Comment (robvet) [reply docs/architecture/cloud-native/infrastructure-resiliency-azure.md:90]: Retracted, but not sure the xref does anything? When I click on them, nothing happens.

File: docs/architecture/cloud-native/resiliency.md (hunk 6 above)
Comment (robvet) [reply docs/architecture/cloud-native/resiliency.md:14]: Fixed.

File: docs/architecture/cloud-native/resilient-communications.md (hunk 7 above)
Comment (robvet) [reply docs/architecture/cloud-native/resilient-communications.md:23]: Addressed

File: docs/architecture/cloud-native/resiliency.md (hunk 5 above)
Comment (robvet) [reply docs/architecture/cloud-native/resiliency.md:12]: Addressed

**Review by nishanil — APPROVED** [review]
Looks good. You still haven't addressed a comment to update the code and images to 3.1. I will not hold this PR for that - instead, we will track this separately.
